backend/Cargo.toml

```diff
@@ -56,6 +56,7 @@ halo2_solidity_verifier = { git = "https://github.com/powdr-labs/halo2-solidity-
   "evm",
 ], optional = true }
 rayon = "1.7.0"
+multiset = "0.0.5"
```

Comment:
Is there really nothing in std that can do this? This crate doesn't even have a minor version, last updated 5 years ago. Not sure about relying on it.

Reply:
Good point! When I added it, I was hoping it would have more capabilities, like multi-set diffing. Just for equality it's easy to implement, will do.